nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -33,7 +60,7 @@ To install over a serial port connect with `115200n8` (e.g.
 `picocom -b 115200 /dev/ttyUSB0`). When the bootloader lists boot
 entries, select the serial console boot entry.
 
-### Networking in the installer {#sec-installation-booting-networking}
+### Networking in the installer {#sec-installation-manual-networking}
 
```

Comment:
Maybe also preserve the old anchor to keep external links working:

```suggestion
### Networking in the installer {#sec-installation-manual-networking}
[]{#sec-installation-booting-networking} <!-- legacy anchor -->

```

Reply:
I thought about that. Thanks for showing how to do it.